example/example.py

```diff
@@ -73,8 +73,8 @@ def show_row(x):
         data = get_db_row(1, 4 + x * row_size, row_size)
         row = snap7.util.DB_Row(data, rc_if_db_1_layout,
                                 layout_offset=4)
-        print 'name', row['RC_IF_NAME']
-        print row['RC_IF_NAME']
+        print(('name', row['RC_IF_NAME']))
```

Comment:
why the double brackets? You can just give multiple arguments to the print functions, now it prints a tuple

Reply:
Fixed